### PR TITLE
Add graph. Refactor about research layout.

### PR DESCRIPTION
**Pull request by dangan114**
graph and chart using d3. not yet implemented responsive layout.

### CODE REVIEW

**Comment by JOOOEG123** [comment]
@dangan114 do not merge this yet. I will merge it when I get home. Can you remove out all the commented code and run prettier. This is a great start. I hope no one else was working on this.

**Review by yzhang54 — APPROVED** [review]

File: src/app/core/services/archives-api-service.ts [hunk 1]
```diff
@@ -15,9 +15,23 @@ export class ArchieveApiService {
...
-    return this.db.object('/persons/requestArchieve/persons').valueChanges();
+    return this.db.object('/persons/requestArchieve/persons').valueChanges()
   }
 
+  getArchiveList() {
+    return this.db.list('/persons/requestArchieve/persons').valueChanges()
+  }
+
+  // getMales() {
+  //   return this.db.list('/persons/requestArchieve/persons',
```
Comment (JOOOEG123) [inline src/app/core/services/archives-api-service.ts:26]: Remove any commented code.

File: src/app/pages/about/chart/chart.component.ts [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+    // Create the Y-axis band scale
+    const y = d3.scaleLinear()
+    .domain([0, Math.max(...data.map(o => o.count))])
+    .range([this.height, 0]);
+
+    // Draw the Y-axis on the DOM
+    this.svg.append("g")
+    .call(d3.axisLeft(y));
+
+    // Create and fill the bars
```
Comment (JOOOEG123) [inline src/app/pages/about/chart/chart.component.ts:92]: Great way to use a comment. This is a custom package so adding a comment is good.

File: src/app/pages/about/chart/chart.component.ts [hunk 3]
```diff
@@ -0,0 +1,178 @@
...
+        return "translate(" + c[0]*2.0 + ", " + c[1]*1.4 + ")"
+      })
+      .style("text-anchor", "middle")
+      .style("font-size", 20);
+
+    
+      // this.svg
+      // .selectAll('pieces')
+      // .data(pie(data))
+      // .enter()
```
Comment (JOOOEG123) [inline src/app/pages/about/chart/chart.component.ts:164]: Remove comment.

File: src/app/app.module.ts [hunk 4]
```diff
@@ -55,6 +55,7 @@ import { LangInterceptor } from './core/interceptors/lang.interceptor';
 import { OverlayComponent } from './pages/gallery/overlay/overlay.component';
 import { ContributionComponent } from './pages/admin/contribution/contribution.component';
 import { RepositoryComponent } from './pages/browse/repository/repository.component';
+import { ChartComponent } from './pages/about/chart/chart.component';
```
Comment (JOOOEG123) [inline src/app/app.module.ts:58]: Move this to the shared components

**Comment by JOOOEG123** [comment]
@dangan114  I will add this. @dangan114  or @weilonp https://stackblitz.com/edit/angular-ivy-r787y7
Can you place the pie with this?

**Pull request merged**